### ISSUE TITLE
Is the project willing to have translation to any other language?

### ISSUE THREAD

**Issue by RonyVidaur** [grzesiek-galezowski/tdd-ebook#92]
I wonder if you consider a good idea to translate the book to **spanish**, if so, please let me know if I can help translating it to spanish.

**Comment by pazthor** [comment]
Hey! I'm native Spanish speaker 
maybe I can help you.  :)

**Comment by grzesiek-galezowski** [comment]
Hi, 

I'm not a spanish speaker, so I don't have the necessary skills to lead this topic, plus there's already a lot of work in the english version. That said, if someone decided to do this and could cope with the idea that the book is meant to be always in motion somehow, it would be great and I can help by answering questions and clarifying meaning of some parts of the text (plus letting know which chapters are failry stable - I'm in the process of revising the already written parts of the book). Plus, the book is on I think most permissive version of Creative Commons, so you can translate, adapt, reuse etc. You can even create your own book based on this book and sell it - the license permits it.

Still, I won't be able to do any part of translation myself or verify whether the translation is true to the original, so this is up to you.

**Comment by RonyVidaur** [comment]
@pazthor I'm native too and that'll be great :) 'cause if we get to do it this will be a great resource for the spanish-speaker community;
however to anyone interested in helping out on this idea, we most emphasize that **it is** and **it will continue to be ** A LOT of work; However I'm not saying I'm going to decide if someone can help or not, as @grzesiek-galezowski this is an open source project and any help is welcome :) 
by the way @grzesiek-galezowski if you could help us to figure out a way to manage this kind of work that'll be awesome, i.e. is it a good idea to have a branch on this project, what would make you more comfortable? because the goal is not making a brand new book based on this, the core idea is to provide the community with a spanish version.

**Comment by grzesiek-galezowski** [comment]
@RonyVidaur it depends on how you plan to distribute it. I use leanpub and you could do the same thing, although from what I read they now charge once per each new book created (I have one book there created before the prices set in that I don't write, I can try to donate by making you co-authors, but that would leave me with access to that book - not sure if it works for you). 

As an alternative, you could use gitbook - this would probably require tweaking markdown syntax since gitbook's dialect is different than leanpub's (this should not be an issue since you would already be changing a lot) and the rendered end result is a little less pretty IMO. Still, this alternative is free AFAIK.

So to sum it up - it's much less about my own comfort as I am open to pretty much every option. It's more about the comfort of the ebook build/distribution system that you intend to use.

**Comment by grzesiek-galezowski** [comment]
You might also have a look at http://alisonstanton.com/gitbook-penflip-leanpub-sphinx-softcover/ - may contain some outdated information though.

**Comment by RonyVidaur** [comment]
@grzesiek-galezowski great article, actually I've been doing some translation since the other day, just to get my hands dirty and I must say I felt comfortable doing it, I'll definetly wanna work on it.
I really think I'm gonna work on this in 2017, so you'll see me around asking you some stuff

**Comment by grzesiek-galezowski** [comment]
Guys, not sure if you are interested in this, but [leanpub has gone freemium](https://leanpub.com/freemium) so creating new books is free once again.

**Comment by grzesiek-galezowski** [comment]
Closing this issue due to no activity in over a year. If for some reason you would like to continue the discussion, please reopen it or open another one.
